yarn-project/end-to-end/src/canary/browser.ts

```diff
@@ -0,0 +1,218 @@
+/* eslint-disable no-console */
+import * as AztecJs from '@aztec/aztec.js';
+import { PrivateTokenContractAbi } from '@aztec/noir-contracts/artifacts';
+
+import { Server } from 'http';
+import Koa from 'koa';
+import serve from 'koa-static';
+import path, { dirname } from 'path';
+import { Browser, Page, launch } from 'puppeteer';
+
+declare global {
+  /**
+   * Helper interface to declare aztec.js within browser context.
+   */
+  interface Window {
+    /**
+     * The aztec.js library.
+     */
+    AztecJs: typeof AztecJs;
+  }
+}
+
+const __filename = AztecJs.fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const PORT = 3000;
+
+const { SANDBOX_URL } = process.env;
+
+const conditionalDescribe = () => (SANDBOX_URL ? describe : describe.skip);
+const privKey = AztecJs.GrumpkinScalar.random();
+
+/**
+ * This test is a bit of a special case as it's relying on sandbox and web browser and not only on anvil and node.js.
```

Comment:
I think it would make more sense to have this comment in the test files and not in the test suite files given that we don't execute this file directly.